Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -372,6 +372,10 @@ void WritePixelShaderCommonHeader(ShaderCode& out, APIType api_type,
             "int3 iround(float3 x) {{ return int3(round(x)); }}\n"
             "int4 iround(float4 x) {{ return int4(round(x)); }}\n\n");
 
+  // GLSL's any() only accepts vector types, not scalar types
+  if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
+    out.Write("bool any(bool b) {{ return b; }}\n");
+
   if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
```

Comment:
These two "if" checks are identical. Is it intentional?

Reply:
The second one has to do with sampler bindings, so I didn't feel it was appropriate to shove this in there.

Reply:
Note that there is another identical if check below that handles uniform bindings, so it's consistent with the existing pattern.